notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        def __call__(self, *args, **kwargs):
</pre><p>if we know names of parameters, can we specify them explicitly for avoiding situations if input signature changed in some new diffusers version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>added explicit arguments</p>